modules/i18n/src/translations/en-US/portals/sms-templates.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { smsTemplatesNS } from "../../../models";
+
+export const smsTemplates: smsTemplatesNS = {
+    dangerZone: {
+        remove: {
+            action: "Remove Template",
+            heading: "Remove Template",
+            message: "This action will remove the selected template and you " +
```

Comment:
Term "template" is kinda repeating, how about "This action will delete the selected template, and any changes you have made will be lost."

Reply:
This was discussed and approved at the design review.